src/main/java/com/company/constants/Constants.java

```diff
@@ -16,7 +16,8 @@ public class Constants {
 
     // PET ERRORS
     public static final String PET_NAME_REQUIRED = "Pet name is required";
-    public static final String PET_GENDER_REQUIRED = "Pet gender is required";
+    public static final String PET_GENDER_REQUIRED = "Pet gender is required and must be valid";
```

Comment:
Yo cambiaría el "and must be valid" ya que al tener dos constantes distintas, se puede diferenciar en cada una. Eso igual manejalo como te parezca. 

Es mas, podes cambiarlo en tu proximo pr del otro enum asi ya mergeas este, como quieras...

Reply:
Dale lo modifico en el proximo PR!!!